src/desktop/apps/gallery_partnerships/client/index.coffee

```diff
@@ -8,6 +8,11 @@ module.exports.init = ->
     $('.gallery-partnerships2__chat__video').hide()
     $('.gallery-partnerships2__chat__fallback_image').show()
 
+  isUserAdmin = sd.CURRENT_USER && sd.CURRENT_USER.type == 'Admin'
```

Comment:
This is more of a style thing but you can also consider the existential operator `?.` here to save some typing:

```coffeescript
isUserAdmin = sd?.CURRENT_USER?.type == 'Admin'
```

Reply:
Nice and concise, I like it 👍